src/network/udp.py

```diff
@@ -153,24 +154,3 @@ def handle_write(self):
             logger.error("socket error on sendato: %s", str(e))
             retval = 0
         self.slice_write_buf(retval)
-
-
-if __name__ == "__main__":
```

Comment:
This is a test. Did you move it somewhere else?

Reply:
It appears to be more debug than a test. Also, it can't possibly be in use because BMConnection is not imported so running it would fail. It appears to be cruft.

Reply:
I looked at it in more detail, and while I'm not fully sure now whether I'd label it a test or debugging, it's originally from elsewhere and unrelated to this file. You're correct to remove it.